webpack.config.js

```diff
@@ -20,10 +20,12 @@ const web = {
       }
     ]
   },
+  node: {
+    fs: 'empty'
```

Comment:
what does this do?

Reply:
Since web doesn't know what `fs` is, webpack has to set it empty to build